guides/common/assembly_administering-hosts.adoc

```diff
@@ -55,3 +55,6 @@ include::modules/proc_displaying-rhel-lifecycle-status.adoc[leveloffset=+2]
 
 include::modules/con_host-search-by-rhel-lifecycle-status.adoc[leveloffset=+2]
 endif::[]
+
+include::modules/proc_viewing-image-mode-hosts-and-booted-container-images.adoc[leveloffset=+1]
+endif::[]
```

Comment:
If image mode hosts are only available with Katello, remove the `endif` above. If they are available without Katello, remove the `endif` below.

Then you don't have to remove it in _master.adoc_, which should stay without changes.